src/quemb/molbe/solver.py

```diff
@@ -908,35 +916,57 @@ def solve_block2(
     mc.fcisolver.twodot_to_onedot = DMRG_args.twodot_to_onedot
     mc.fcisolver.maxIter = DMRG_args.max_iter
     mc.fcisolver.block_extra_keyword = DMRG_args.block_extra_keyword
-    mc.fcisolver.scratchDirectory = frag_scratch.path
-    mc.fcisolver.runtimeDir = frag_scratch.path
+    mc.fcisolver.scratchDirectory = scratch
+    mc.fcisolver.runtimeDir = scratch
     mc.fcisolver.memory = DMRG_args.max_mem
-    os.chdir(frag_scratch)
 
-    mc.kernel(orbs)
-    rdm1, rdm2 = dmrgscf.DMRGCI.make_rdm12(
-        mc.fcisolver, DMRG_args.root, DMRG_args.norb, DMRG_args.nelec
-    )
+    if DMRG_args.force_earlystop:
+        # Generates the input files for block2 but stops short of calling it.
+        # Instead loads the 1- and 2-RDM from `scratch` and proceed with BE.
+        # (Currently this is only used in the unit tests: `dmrg_molBE_test`)
+        dmrgscf.dmrgci.writeDMRGConfFile(mc.fcisolver, DMRG_args.nelec, Restart=False)
+        rdm1_path = Path(scratch + "/rdm1.npy")
```

Comment:
`Path(scratch + "/rdm1.npy")`
is much nicer written as 
`frag_scratch / "rdm1.npy"`

(the type cast to `Path` is fully redundant)

Reply:
BTW: If you cast to be sure that you indeed get a specific type, I would add a type specifier, i.e.
```python3
rdm1_path: Path = ...
`` `